README.md

```diff
@@ -21,53 +21,67 @@ mode and editor commands, making the best use of both editors.
+    -   [Code navigation bindings](#code-navigation-bindings)
+    -   [Explorer/list navigation bindings](#explorerlist-navigation-bindings)
+    -   [Explorer file manipulation bindings](#explorer-file-manipulation-bindings)
+    -   [Hover widget manipulation bindings](#hover-widget-manipulation-bindings)
+-   [📟 Neovim Commands](#-neovim-commands)
+    -   [File management](#file-management)
+    -   [Tab management](#tab-management)
+    -   [Buffer/window management](#bufferwindow-management)
+-   [🎨 Highlights](#-highlights)
 -   [🧰 Developing](#-developing)
 -   [❤️ Credits \& External Resources](#️-credits--external-resources)
 
-</details>
-
 ## 🧰 Getting Started
 
 ### Installation
 
--   Install the [vscode-neovim](https://marketplace.visualstudio.com/items?itemName=asvetliakov.vscode-neovim)
-    extension.
--   Install [Neovim](https://github.com/neovim/neovim/wiki/Installing-Neovim) **0.9.0** or greater (0.10 recommended,
-    [see below](#neovim-configuration-for-best-compatibility)).
-    -   Set the Neovim path in the extension settings. You must specify the full path to Neovim, like
-        "`C:\Neovim\bin\nvim.exe`" or "`/usr/local/bin/nvim`".
-    -   The setting id is "`vscode-neovim.neovimExecutablePaths.win32/linux/darwin`", respective to your system.
--   If you want to use Neovim from WSL, set the `useWSL` configuration toggle and specify the Linux path to the nvim
-    binary. `wsl.exe` Windows binary and `wslpath` Linux binary are required for this. `wslpath` must be available
-    through `$PATH` Linux env setting. Use `wsl --list` to check for the correct default Linux distribution.
--   Assign [affinity](#affinity) value for performance improvement.
+Install the [vscode-neovim](https://marketplace.visualstudio.com/items?itemName=asvetliakov.vscode-neovim) extension.
 
-    -   Go to Settings > Features > Extensions > Experimental Affinity.
+Install [Neovim](https://github.com/neovim/neovim/wiki/Installing-Neovim) **0.9.0** or greater (0.10 recommended).
 
-        Add an entry with item name `asvetliakov.vscode-neovim` and value 1.
+> **Note:** Though the extension strives to be as compatible as possible with older versions of Neovim, some older
+> versions may have quirks that are not present anymore. In light of this, certain configuration settings are
+> recommended in some older versions for the best experience. These can be found
+> [on the wiki](https://github.com/vscode-neovim/vscode-neovim/wiki/Version-Compatibility-Notes).
 
-        OR
+\[Optional\] Set the Neovim path in the extension settings under
+"`vscode-neovim.neovimExecutablePaths.win32/linux/darwin`", respective to your system. For example,
+"`C:\Neovim\bin\nvim.exe`" or "`/usr/local/bin/nvim`".
 
-    -   Add to your `settings.json`:
-        ```json
-        "extensions.experimental.affinity": {
-            "asvetliakov.vscode-neovim": 1
-        },
-        ```
```

Comment:
@theol0403 
Is this setting no longer necessary?

Reply:
We add it automatically on setup